Documentation/In-depth/TheInstallTool/Index.rst

```diff
@@ -94,6 +94,10 @@ description of the settings carefully at least once, so you get an
 impression of what you can configure. Normally you *can*, but you *don't
 have to* change anything here during installation, as the previous steps
 took care of the most important settings.
+Pay attention to use a https website and set the BE loginSecurityLevel to normal.
```

Comment:
```suggestion
Pay attention to use HTTPS for your website and set :code:`$GLOBALS['TYPO3_CONF_VARS']['BE']['loginSecurityLevel']` to `normal`.
```

This is not exactly true. Even with HTTPS you can use RSAAuth, it's just pointless. Thus we should rephrase this to a recommendation.